### ISSUE TITLE
Delete empty SwiftSupport and Framework folders at the archive task

### ISSUE THREAD

**Issue by BamX** [openbakery/gradle-xcodePlugin#242]
There a problem with empty SwiftSupport folder when I publish an application to AppStore.
> Invalid Swift Support - The SwiftSupport folder is empty. Rebuild your app using the current public (GM) version of Xcode and resubmit it.

It's critical for me, I can't publish build.

**Comment by renep** [comment]
I have the same issue with an empty Frameworks folder in one of my projects. The xcodebuild command creates this, but does not remove it when this is empty.

You can simply create a workaround for this by creating a task that is depended to archive and deletes the empty directory. e.g.:
```
task('myArchive, dependsOn:['archive']) {
				doLast {
					println "Delete empty frameworks directory"
					new File('build/archive/<CHANGE_ME!!!!>xcarchive/Products/Applications/<CHANGE_ME!!!!>/SwiftSupport/').delete()  
				}
			}
```

When you then execute `gradle myArchive` then the archive is created and then the empty SwiftSupport is deleted.

I will adress this in the next release, so that you do not need the workaround anymore.

At least with gradle you have the power to do such workarounds ;)

**Comment by BamX** [comment]
Thank you! Here code I used as workaround:
```
task removeEmptyFrameworksDir(dependsOn: 'xcodebuild') {
    description = "Remove empty Frameworks dir from build results"
    doFirst {
      new File('build/sym/').eachFileRecurse (FileType.DIRECTORIES) { appFile ->
        if (appFile.name.endsWith(".app")) {
          appFile.eachFile (FileType.DIRECTORIES) { frameworksDir ->
            if (frameworksDir.name == 'Frameworks') {
              def empty = frameworksDir.list().length == 0
              if (empty) {
                  frameworksDir.deleteDir()
                  logger.info("Removed ${frameworksDir.absolutePath}")
              }
            }
          }
        }
      }
    }
  }
  tasks['archive'].dependsOn(removeEmptyFrameworksDir)
```

**Comment by renep** [comment]
fixed with 0.13.0
